### PR TITLE
Roll tonic to pick up my crash fixes

### PR DESCRIPTION
**Pull request by Hixie**
Also fix the license script to not use "lib" as a library name.

### CODE REVIEW

**Comment by Hixie** [comment]
oh i need to update the master license file also.

**Comment by Hixie** [comment]
cc @jason-simmons for review

File: sky/packages/sky_engine/LICENSE [hunk 1]
```diff
@@ -10005,6 +10005,128 @@ Legal Terms
 
 --- end of FTL.TXT ---
 --------------------------------------------------------------------------------
+ftl
+
+Copyright 2016 The Fuchisa Authors. All rights reserved.
```
Comment (jason-simmons) [inline sky/packages/sky_engine/LICENSE]: typo: "Fuchsia"

(I assume this originated in some FTL source file?)

Comment (Hixie) [inline sky/packages/sky_engine/LICENSE]: Thanks, I'll report it upstream.

**Pull request merged**